circuit_puzzles/programs/announcer_configure.clsp

```diff
@@ -0,0 +1,88 @@
+; governance backdoor
+; we either activate this announcer or deactivate, can happen at any time via CRT vote
+(mod ((MOD_HASH STATUTES_STRUCT LAUNCHER_ID INNER_PUZZLE_HASH APPROVED DEPOSIT DELAY ATOM_VALUE COOLDOWN_START TIMESTAMP_EXPIRES)
+      ; solution
+      inner_puzzle_hash new_puzzle_hash deposit input_conditions
+      (current_timestamp statutes_inner_puzzle_hash input_toggle_activation deactivation_cooldown_interval delay new_price melt)
```

Comment:
rename new_price to new_atom_value to keep terminology consistent with other files?